spec/rubocop/cli_spec.rb

```diff
@@ -2285,8 +2285,7 @@ def method(foo, bar, qux, fred, arg5, f) end #{'#' * 85}
       create_file('example.rb', <<~RUBY)
         # frozen_string_literal: true
 
-        if foo and bar
-        end
+        a.foo..b.bar # Lint/AmbiguousRange offense is not safe correctable
```

Comment:
Is this an intentional change?

Reply:
Yes, because `Lint/EmptyConditionalBody` is no longer `SafeAutoCorrect: false`, the test was failing so I replaced it with a different `SafeAutoCorrect: false` cop.